File: Java/KodeProgram/Elektronik.java
// Elektronik.java
// Berisi definisi class abstrak Elektronik, turunan dari Produk.
abstract class Elektronik extends Produk {
    private String brand;
    private int garansi;
    private int stok;

    public Elektronik(int id, String nama, double harga,
                      String brand, int garansi, int stok) {
        super(id, nama, harga);
        this.brand = brand;
        this.garansi = garansi;
        this.stok = stok;
    }

    public String getBrand() { return brand; }
    public int getGaransi() { return garansi; }
    public int getStok() { return stok; }
}

File: Python/KodeProgram/Elektronik.py
# Elektronik.py
# Turunan Level 2: Elektronik
from Produk import Produk

class Elektronik(Produk):
    def __init__(self, id, nama, harga, brand, garansi, stok):
        super().__init__(id, nama, harga)
        self.__brand = brand
        self.__garansi = garansi
        self.__stok = stok

    def get_brand(self):
        return self.__brand

    def get_garansi(self):
        return self.__garansi

    def get_stok(self):
        return self.__stok